src/Razor/Razor.Runtime/ref/Microsoft.AspNetCore.Razor.Runtime.Manual.cs

```diff
@@ -1,23 +1,12 @@
 // Copyright (c) .NET Foundation. All rights reserved.
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
-using System.Runtime.CompilerServices;
-using Microsoft.AspNetCore.Razor.TagHelpers;
-
-[assembly: TypeForwardedTo(typeof(DefaultTagHelperContent))]
-[assembly: TypeForwardedTo(typeof(HtmlAttributeNameAttribute))]
-[assembly: TypeForwardedTo(typeof(HtmlAttributeNotBoundAttribute))]
-[assembly: TypeForwardedTo(typeof(HtmlTargetElementAttribute))]
-[assembly: TypeForwardedTo(typeof(ITagHelper))]
-[assembly: TypeForwardedTo(typeof(ITagHelperComponent))]
-[assembly: TypeForwardedTo(typeof(NullHtmlEncoder))]
-[assembly: TypeForwardedTo(typeof(OutputElementHintAttribute))]
-[assembly: TypeForwardedTo(typeof(ReadOnlyTagHelperAttributeList))]
-[assembly: TypeForwardedTo(typeof(RestrictChildrenAttribute))]
-[assembly: TypeForwardedTo(typeof(TagHelper))]
-[assembly: TypeForwardedTo(typeof(TagHelperAttribute))]
-[assembly: TypeForwardedTo(typeof(TagHelperAttributeList))]
-[assembly: TypeForwardedTo(typeof(TagHelperComponent))]
-[assembly: TypeForwardedTo(typeof(TagHelperContent))]
-[assembly: TypeForwardedTo(typeof(TagHelperContext))]
-[assembly: TypeForwardedTo(typeof(TagHelperOutput))]
\ No newline at end of file
+namespace Microsoft.AspNetCore.Razor.Runtime.TagHelpers
+{
+    public partial class TagHelperExecutionContext
+    {
+        internal TagHelperExecutionContext(string tagName, Microsoft.AspNetCore.Razor.TagHelpers.TagMode tagMode) { }
+        [System.Diagnostics.DebuggerStepThroughAttribute]
+        internal System.Threading.Tasks.Task<Microsoft.AspNetCore.Razor.TagHelpers.TagHelperContent> GetChildContentAsync(bool useCachedResult, System.Text.Encodings.Web.HtmlEncoder encoder) { throw null; }
+    }
+}
```

Comment:
@dougbu So far this is the only changed (as apposed to added or deleted) Manual.cs file in the merge. It seems like this change was made in release/3.1. Is this expected?

@JunTaoLuo

Reply:
Best to undo all changes to *.Manual.cs files then regenerate the ref/ projects (again). We don't need the `internal` APIs in 'master' and including them will make everything much more brittle.

Reply:
I reverted this file to master and re-ran GenerateReferenceAssemblies.ps1.